Documentation/Signoffs/Heartbeat Respiratory Subsystem.md

```diff
@@ -0,0 +1,112 @@
+# Heartbeat/Respiratory Subsystem 
+
+## Subsystem Function
+
+The purpose of this subsystem is to measure the presence of a heartbeat as well as measure the respiratory rate of a person contactlessly. This subsystem will be composed of a singular radar that will perform both of these actions. The measuring of a person’s presence of a heartbeat as well as respiratory rate will be done by measuring the rise and fall rates of a person’s chest to measure the respiratory rate, and the rise and fall rates of the area around the heart to measure the presence of a heartbeat. A series of signals will be sent and received to measure these changes. The raw data will be put through an amplifier/filter and then be converted to a digital signal that is sent to a computing system to be put through a processing algorithm,
+
+## Specs and Constraints			
+
+
+| Description | Constraint | Source of Constraint |
+|-------------|------------|----------------------|
+| Heartbeat analysis | Must sense Beats per minute from 27 to 200 bpm (or frequencies from 0.45 Hz to 3.33 Hz) | From Conceptual Design, from Heartbeat/Respiratory System section |
+| Respiratory sensor | Must sense breaths per minute from 7.8 to 30 bpm (or frequencies from 0.13 Hz to 0.50 Hz) | From Conceptual Design, from Heartbeat/Respiratory System section |
+| Distance | Must detect and measure heartbeat and breath rate from at least 1 meter away | From DARPA constraints |
+| Weight | Entire system must weigh less than 6 pounds, this subsystem should weight 1.5 lbs or less | From DARPA constraints/ Conceptual design |
+| Safety | Radar must not emit at a frequency over 10 GHz with a power density of 1000 W/m^2 in order to keep the radar skin and eye safe. | From World Health Organization (WHO) |
+
+
+## Wiring Schematics
+
+
+
+![image](https://user-images.githubusercontent.com/79685126/232992358-859ff652-7a55-444c-bf29-dbeecf625869.png)
+
+
+Figure1. Wire Diagram for the total subsystem
+
+
+
+![image](https://user-images.githubusercontent.com/79685126/232992308-61d8f565-3325-4c40-90d2-4f7287ad85ea.png)
+
+
+Figure 2. Circuit schematic for active amplifier
+
+## Analysis:
+
+1+2.) The 24 GHz doppler  radar functions by transmitting a signal described by the equation T(t) = Acos[2pift + theta(t)] This then hits the target and returns the function listed below.
```

Comment:
This is good. What it does right is show the general idea. But what it doesn't do is show that you should be able to recover the hr and br for the range you have specified.